jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/WorkspaceConverterMojo.java

```diff
@@ -154,12 +160,12 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 
     PackageJsonPrettyPrinter prettyPrinter = new PackageJsonPrettyPrinter();
 
-    objectMapper = SenchaUtils.getObjectMapper()
+    objectMapper = new ObjectMapper()
             .setDefaultPrettyPrinter(prettyPrinter)
+            .configure(JsonParser.Feature.ALLOW_COMMENTS, true)
```

Comment:
Do we really need that?

Reply:
probably not, since we only use the ObjectMapper only for parsing the package.json, app.json and appmanifests.